tests/test_style.py

```diff
@@ -106,7 +106,7 @@ def test_include_styles_overriding_values(offline, tmp_path):
         "styles/isort2",
         f"""
         [nitpick.styles]
-        include = ["styles/isort2.toml", "flake8.toml"]
```

Comment:
This behaviour is intended: "styles/isort2.toml" is here on purpose, to test that the code doesn't fail with a recursion error.

I added a reminder it to [the docstring above](https://github.com/andreoliwa/nitpick/pull/333/files#diff-33a1ad85feedd65d581c754383eb79e2f185e873775fc64194d1c7c35dcefe20R94):

```python
"""One style file can include another (also recursively). Ignore styles that were already included."""
```

A suggestion to restore the behaviour:

1. Resolve the absolute name of the style;
2. Ignore a style that was already included, and don't include it again.

---

The intended behaviour is not very clear because this test is actually testing 2 things.
A better approach could have been to create 2 separate tests:

- `test_include_styles_overriding_values()`
- `test_ignore_styles_that_were_already_included()` or `test_dont_raise_recursion_error_on_duplicated_styles()`

This was a bad design on my side.
No need to split into 2 tests now in this pull request.
This can be improved later.